src/extension/ln/alby/index.ts

```diff
@@ -0,0 +1,79 @@
+export default class AlbyProvider {
+  enabled: boolean;
+
+  constructor() {
+    this.enabled = false;
+  }
+
+  enable() {
+    if (this.enabled) {
+      return Promise.resolve({ enabled: true });
+    }
+    return this.execute("enable").then((result) => {
+      if (typeof result.enabled === "boolean") {
+        this.enabled = result.enabled;
+      }
+      return result;
+    });
+  }
+
+  /**
+   * Adds a wallet to the user's Alby extension
+   *
+   * @param name The name of the account
+   * @param connector The connector to use
+   * @param config The config to pass to the connector
+   * @returns Nothing, throws if user rejects
+   */
+  addAccount(name: string, connector: string, config: Record<string, unknown>) {
```

Comment:
Is the name of the account something that the user/app will always have to provide? or could it be an optional parameter?

Reply:
I was also thinking about letting the user enter the name of the account. But that could also be added later on maybe? :thinking: